Resources/Prototypes/Entities/Structures/Machines/lathe.yml

```
@@ -796,7 +801,10 @@
       - GrenadeFlashBang # Goobstation
       - BolaEnergy # Goobstation
       - CartridgeEpinephrine # Goobstation
-      - SyringeGun #Goobstation
+      - CartridgeBicaridine # Goobstation
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)